src/containers/Tenant/utils/schema.ts

```diff
@@ -1,10 +1,23 @@
-import type {NavigationTreeNodeType} from "ydb-ui-components";
+import type {NavigationTreeNodeType} from 'ydb-ui-components';
+import {EPathType} from '../../../types/api/schema';
 
-const DB_TYPES = new Set(['EPathTypeSubDomain']);
-const TABLE_TYPES = new Set(['EPathTypeTable', 'EPathTypeOlapTable']);
-const DIR_TYPES = new Set(['EPathTypeDir', 'EPathTypeOlapStore']);
+const DB_TYPES = new Set([
+    EPathType.EPathTypeSubDomain,
+]);
+const TABLE_TYPES = new Set([
+    EPathType.EPathTypeTable,
+    EPathType.EPathTypeColumnTable,
+]);
+const DIR_TYPES = new Set([
+    EPathType.EPathTypeDir,
+    EPathType.EPathTypeColumnStore,
+]);
+
+export const calcNavigationTreeType = (type?: EPathType): NavigationTreeNodeType => {
+    if (!type) {
+        return 'directory';
+    }
 
-export const calcNavigationTreeType = (type: string): NavigationTreeNodeType => {
     if (DIR_TYPES.has(type)) {
         return 'directory';
```

Comment:
It seems it would be useful to add types also for returning value to improve extensibility.

Reply:
The return value type is determined by the lib type `NavigationTreeNodeType`, and it is a union of strings, I don't think adding our own type would help. By the contrary, this would add another entity to modify in case there are updates in the lib.

Actually it is maybe better to rename the function to something like `mapPathTypeToNavigationTreeType`. I'll think about that.